python_modules/dagster/dagster/_core/storage/event_log/schema.py

```diff
@@ -137,6 +137,25 @@
     db.Column("create_timestamp", db.DateTime, server_default=get_current_timestamp()),
 )
 
+AssetChecksTable = db.Table(
+    "asset_checks",
```

Comment:
Should we call this "asset_check_runs"? The name "asset_checks" would lead me to believe that this would have a row per defined check, which isn't the case if I understand correctly.